package.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
What is this package.json for? The overall project? I believe that we can get rid of this one since each section should have their own package json.